src/test/java/com/powsybl/openloadflow/network/AbstractLoadFlowNetworkFactory.java

```diff
@@ -63,7 +63,7 @@ protected static Generator createGenerator(Bus b, String id, double p, double v)
                 .setConnectableBus(b.getId())
                 .setEnergySource(EnergySource.OTHER)
                 .setMinP(0)
-                .setMaxP(p)
+                .setMaxP(2 * p)
```

Comment:
Why 2 times p and not... 10 times p?
If we wanted to be sure that slack will be distributed, couldn't we set maxP after the whole network is built to something like maxP = sum(targetPs + loadP0s)?

Reply:
Yes, it is a choice and I don't know if it is relevant. As you can see, in some tests it is not enough. @jeandemanged any opinion?

Reply:
2* is enough, I'd say it is the unit test's author responsibility to verify it's test is **really** a **unit** test, i.e. not relying too much of many OLF features within a single test.